bin/sign-images.py

```diff
@@ -138,7 +138,9 @@ def main():
             except (FileNotFoundError, json.JSONDecodeError):
                 print(f"Error: Could not find or parse local file {json_file}")
                 sys.exit(1)
-
+        subprocess.run(
+            ["docker", "login", "quay.io", "-u", os.environ["QUAY_USERNAME"], "-p", os.environ["QUAY_PASSWORD"]], check=True
+        )
```

Comment:
This should be a prerequisite step, not included in this script. This script signs the images and publishes, regardless of what registry that image is held in.